.github/workflows/build.yml

```diff
@@ -0,0 +1,11 @@
+name: Deploy to Cloudflare
+
+on:
+  - push
```

Comment:
I think on merge to main we want a release and on create Pull Request we only need a preview.

I'm not sure how it works with Cloudflare, but in netlify the PR part of that will happen automatically once the repo is wired up.

Reply:
i am going to look into making this only deploy on modifications to main. is that enough? as this is a super prototype alpha project do we need to worry about secure production branches?